v1-to-v2-data-migration/countryData/addressMappings.ts

```diff
@@ -27,7 +31,7 @@ export const ADDRESS_MAPPINGS: Record<string, AddressConfigFunction> = {
     'child.birthLocation.privateHome': { streetLevelDetails: { street: data } },
   }),
   'birth.child.addressLine3Placeofbirth': (data: string) => ({
-    'child.birthLocation.privateHome': { streetLevelDetails: { number: data } },
+    'child.birthLocation.privateHome': { streetLevelDetails: { number: clearUrbanRuralValue(data) } },
```

Comment:
In many cases the word "URBAN" or "RURAL" ended up flowing to the "number" field of address. I now made it so if the value we assume to be the number happens to be "URBAN" or "RURAL", we just leave it empty

<img width="818" height="249" alt="Image" src="https://github.com/user-attachments/assets/30add46c-ee47-4b04-9331-1986dfc11823" />